lib/VisualBuilder.js

```diff
@@ -143,6 +144,38 @@ class VisualBuilder extends EventEmitter {
         this._resetWatchStates();
     }
 
+    /**
+     * Validates the visual author name, email, visual description and visual support URL were specified
+     * 
+     * @param {VisualPackage} visualPackage - instance of a visual package to validate 
+     * @private
+     */
+    _validateVisualMetaInformation(visualPackage) {
```

Comment:
Unit tests must be updated as well since some of them don't specify all of required metadata that the new code checks.